test/Header.test.js

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
-import { mount } from 'enzyme';
-import Header from "../src/components/Header";
-import configureMockStore from 'redux-mock-store';
-
-const initialState = {
-  search: {
-    searchby: 'TITLE',
-    phrase: ''
-  }
-};
-
-const mockStore = configureMockStore();
+import { shallow, mount } from 'enzyme';
+import { Header } from "../src/components/Header";
 
 describe('Header', () => {
-  const store = mockStore(initialState);
   it('should render Header component', () => {
     const component = mount(
       <Header searchby={'TITLE'}
-        handleSearchByClick={jest.fn()}
-        store={store} />);
+        handleSearchByClick={jest.fn()} />);
     expect(component).toMatchSnapshot();
   });
 });
+
+it('should call prop method', () => {
+  const spy = jest.fn();
+  const eventMock = {
+    target: {
+      value: 'test'
+    }
+  };
+
+  const instance = shallow(<Header handleChange={spy} searchMovieChange={spy} />).instance();
```

Comment:
Using the same spy in multiple places makes it unreliable - we don't know who called the spy.

Reply:
You are right. I've just fixed that.